src/reference/sectionA_definitions.inc

```diff
@@ -134,7 +134,7 @@ Semantically equivalent CellML infosets
 
 #. Two :ref:`CellML infosets<specA_cellml_infoset>` SHALL be deemed semantically equivalent if one can be transformed into the other by making any or none of the following changes:
 
-   #. Adding, removing, and/or modifying comment information items.
+   #. Adding, removing, and/or modifying comment information items (https://www.w3.org/TR/xml-infoset/#infoitem.comment).
```

Comment:
I would make "comment information items" a link.

Reply:
good idea, done that.